File: backend/src/models/producto.model.js


const { Schema, model } = require('mongoose');
const productoSchema = new Schema({
   
    fecha: String,
    cliente: String,
    nombre: String,
    code: String,
    cantidad: String,
    posicion: String,
    
})

module.exports = model('Producto', productoSchema);

File: proceso_pza/lib/modelos/producto.model.dart
class Productos {
  var id;
  var name;
  var code;
  var amount;
  var position;
  var date;
  var client;

  Productos({
    this.id,
    this.name,
    this.code,
    this.amount,
    this.position,
    this.date,
    this.client,
  });

  factory Productos.fromJson(Map<String, dynamic> json) {
    return Productos(
        id: json['_id'],
        name: json['nombre'],
        code: json['code'],
        amount: json['cantidad'],
        position: json['posicion'],
        date: json['fecha'],
        client: json['cliente']);
  }
}
